src/fastoad_cs25/configurations/cs25_base.yaml

```diff
@@ -41,6 +41,8 @@ model:
       id: fastoad.handling_qualities.static_margin
     wing_position:
       id: fastoad.loop.wing_position
+    chord_global_positions:
+      id: fastoad.geometry.global_chord_positions
```

Comment:
It is quite compelling to have this "small" model at the system level.
I tried to move it to `geometry.py` even though it adds a feedback loop on the wing position (that is already present anyway). The computation is very very slightly increased (0.01 s). What do you think? This would be more compact and natural to have it in the geometry module.